pkg/driver/bacnet/merge/emergency.go

```diff
@@ -128,6 +137,26 @@ func (t *emergencyImpl) pollPeer(ctx context.Context) (*traits.Emergency, error)
 		})
 	}
 
+	if t.config.AlarmConfig != nil {
+		requestNames = append(requestNames, "alarmConfig")
+		readValues = append(readValues, t.config.AlarmConfig.ValueSource)
+		resProcessors = append(resProcessors, func(response any) error {
+			enum, err := comm.EnumValue(response)
```

Comment:
I can't help thinking this is going to surprise us later. There's no indication from the config names (or doc) that we expect the value of AlarmConfig to be an enum type. What's the reason for this? Can we rename AlarmConfig to `Enum` or something? Can we loosen this line so it can read analog or binary values as well as enums? etc.